Assignment-07-Product Designer/ui-training-assignment-7.htm

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Assignment 7</title>
+    <link rel="stylesheet" href="styles/css/assignment7.css" />
+    <script
+      src="https://kit.fontawesome.com/e58997d292.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles/css/assignment7.css" />
+    <style>
+      /*imported google fonts style for roboto font*/
+      @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap");
+    </style>
+  </head>
+  <body>
+    <!--parent-container-->
+    <div class="parent_container">
+      <!--initial email header to be displayed at flex end on the top-->
+      <article class="initial_header">
+        <section class="email">shrutti.bhatt@siriuscom.com</section>
+      </article>
+      <!--end-->
+
+      <!--Shrutti designation and role-->
+      <article class="flex_container_shrutti">
+        <section class="shrutti">
```

Comment:
please use valid class names for html and css

Reply:
review completed:
- shrutti class name changed to 'email_contianer'.
- 'shrutti first name' changed to 'user_first_name'.